src/clear-refinements/clear-refinements.ts

```diff
@@ -16,19 +16,21 @@ import { noop } from '../utils';
         (click)="handleClick($event)"
         [disabled]="!state.hasRefinements"
       >
-        {{buttonLabel}}
+        {{resetLabel}}
       </button>
     </div>
   `,
 })
 export class NgAisClearRefinements extends BaseWidget {
-  @Input() public buttonLabel: string = 'Clear refinements';
-  @Input() public clearsQuery: boolean = false;
-  @Input() public excludeAttributes: string[] = [];
+  @Input() public resetLabel: string = 'Clear refinements';
+  @Input() public includedAttributes: string[] = [];
+  @Input() public excludedAttributes: string[] = [];
```

Comment:
The default value will override the connectors default value which is `['query']`. Note that both `includedAttributes` & `excludedAttributes ` should be optional.

Reply:
you're right, i've removed them altogether Thanks!